relay-server/src/services/processor/playstation.rs

```diff
@@ -0,0 +1,40 @@
+//! Playstation related code.
+//!
+//! These functions are included only in the processing mode.
+
+use relay_dynamic_config::Feature;
+use relay_event_schema::protocol::Event;
+use relay_protocol::Annotated;
+
+use crate::envelope::{AttachmentType, ItemType};
+use crate::services::processor::metric;
+use crate::services::processor::{ErrorGroup, EventFullyNormalized, ProcessingError};
+use crate::services::projects::project::ProjectInfo;
+use crate::statsd::RelayCounters;
+use crate::utils::TypedEnvelope;
+
+pub fn expand(
+    managed_envelope: &mut TypedEnvelope<ErrorGroup>,
+    project_info: &ProjectInfo,
+) -> Result<(), ProcessingError> {
+    let envelope = &mut managed_envelope.envelope_mut();
+
+    if let Some(_item) = envelope.take_item_by(|item| {
+        item.ty() == &ItemType::Attachment
+            && item.attachment_type() == Some(&AttachmentType::Prosperodump)
+            && project_info.has_feature(Feature::PlaystationIngestion)
```

Comment:
Can just check the feature flag at the beginning of the fn and return early, makes this a bit shorter.